xbmc/pvr/windows/GUIWindowPVRBase.h

```diff
@@ -84,7 +84,7 @@ namespace PVR
 
     bool InitChannelGroup(void);
     virtual CPVRChannelGroupPtr GetChannelGroup(void);
-    virtual void SetChannelGroup(const CPVRChannelGroupPtr &group);
+    void SetChannelGroup(const CPVRChannelGroupPtr &group, bool bUpdate = true);
```

Comment:
Do you mind adding a docblock so the purpose of the new boolean becomes clearer?